src/content/energy-consumption/index.md

```diff
@@ -8,40 +8,43 @@ lang: en
-| PoW Ethereum          |                 78                  |          30,000x           | [source](https://digiconomist.net/ethereum-energy-consumption)                                                                                                                    |
-| Youtube (direct only) |                 12                  |           4600x            | [source](https://www.gstatic.com/gumdrop/sustainability/google-2020-environmental-report.pdf)                                                                                     |
-| Gaming in USA         |                 34                  |          13,000x           | [source](https://www.researchgate.net/publication/336909520_Toward_Greener_Gaming_Estimating_National_Energy_Use_and_Energy_Efficiency_Potential)                                 |
-| Netflix               |                0.451                |            173x            | [source](https://assets.ctfassets.net/4cd45et68cgf/7B2bKCqkXDfHLadrjrNWD8/e44583e5b288bdf61e8bf3d7f8562884/2021_US_EN_Netflix_EnvironmentalSocialGovernanceReport-2021_Final.pdf) |
-| PayPal                |                0.26                 |            100x            | [source](https://app.impaakt.com/analyses/paypal-consumed-264100-mwh-of-energy-in-2020-24-from-non-renewable-sources-27261)                                                       |
-| AirBnB                |                0.02                 |             8x             | [source](<https://s26.q4cdn.com/656283129/files/doc_downloads/governance_doc_updated/Airbnb-ESG-Factsheet-(Final).pdf>)                                                           |
-| PoS Ethereum          |               0.0026                |             1x             | [source](https://carbon-ratings.com/eth-report-2022)                                                                                                                              |
+|                     | Annualized energy consumption (TWh) | Comparison to PoS Ethereum |                                                                                      Source                                                                                       |
+| :------------------ | :---------------------------------: | :------------------------: | :-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------: |
+| Banking system      |                 239                 |          92,000x           |                                                                [source](https://docsend.com/view/adwmdeeyfvqwecj2)                                                                |
```

Comment:
This feels a little fuzzy to me, it's too vague an example to wrap your head around. 

Fine with other changes, but I could see this particular example adding confusion... do you think we lose much by removing it @dmatsuoka?

Reply:
That's fine with me. We can always revisit new sources down the road again as they become available. I've removed it.